cmd/lint-panics/analyzer/analyzer.go

```diff
@@ -0,0 +1,247 @@
+package analyzer
+
+import (
+	"context"
+	"fmt"
+	"go/ast"
+	"os"
+	"time"
+
+	"go.uber.org/zap"
+
+	goparser "go/parser"
+	gotoken "go/token"
+	"strings"
+
+	"github.com/pkg/errors"
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+
+	gopls2 "github.com/status-im/status-go/cmd/lint-panics/gopls"
```

Comment:
```suggestion
	gopls2 "github.com/status-im/status-go/cmd/lint-panics/gopls"
```

Naming with numbers makes my eyes bleed :D

Reply:
lol, but what's your proposal? 😄 
<img width="595" alt="image" src="https://github.com/user-attachments/assets/c92bba27-1740-4582-9339-e977684b35cc">